packages/generator/src/action-function-import/action.ts

```diff
@@ -7,15 +7,29 @@ import { additionalDocForEntityNotDeserializable } from './function';
 
 const parameterName = 'parameters';
 
+function actionImportReturnType(actionImport: VdmActionImport) {
+  let type = actionImport.returnType.returnType;
+
+  if (actionImport.returnType.isCollection) {
+    type = `${type}[]`;
+  }
+  if (actionImport.returnType.isNullable) {
+    type = `${type}|null`;
+  }
```

Comment:
Minor, but for better readability:
```suggestion
  if (actionImport.returnType.isCollection) {
    type = `${type}[]`;
  }
  
  if (actionImport.returnType.isNullable) {
    type = `${type} | null`;
  }
```